features/bootstrap/MfaContext.php

```diff
@@ -582,7 +541,7 @@ public function theUsersBrowserSupportsUf()
             'Update USER_AGENT_WITH_WEBAUTHN_SUPPORT to a User Agent with WebAuthn support'
         );
         
-        $this->driver->getClient()->setServerParameter('HTTP_USER_AGENT', $userAgentWithWebAuthn);
+//        $this->driver->getClient()->setServerParameter('HTTP_USER_AGENT', $userAgentWithWebAuthn);
```

Comment:
This seems like a significant change... I think that (now commented-out) line was what set the user agent to a string expected to indicate WebAuthn support. Was this change intended to be temporary?

Reply:
Yes, it is temporary. I spent many long and frustrating hours trying to find a browser emulator driver that would support this. As you have seen in the feature file, all of the tests that simulate a browser that does not support webauthn have been disabled.

The longer answer is that the original Mink driver that was used for these tests, Goutte, has been archived in favor of Browser Kit. I was unable to get that to work -- the only documentation I could find was for other frameworks. The Chrome driver doesn't seem to have a way to simulate a different user agent.

Reply:
Okay, thanks. I'm sorry it took so many frustrating hours to sort through that.

Since it looks like lack-of-WebAuthn-support is becoming a thing of the past, we could eventually probably remove a lot of this (tests and code) that is focused on detecting WebAuthn support. (Later, not needed right now, and probably after some more discussion.)

Thanks again for your work on this.